opencti-platform/opencti-front/src/private/components/common/files/CustomFileUploader.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FormEvent, useState } from 'react';
+import { styled } from '@mui/material/styles';
+import { FormikErrors } from 'formik';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { useFormatter } from '../../../../components/i18n';
+import { truncate } from "../../../../utils/String";
+import {Theme} from "@mui/material/styles/createTheme";
+import makeStyles from "@mui/styles/makeStyles";
```

Comment:
Can you reformat this part to align with ESlint rules ?
Strings must use singlequote and a space is required after '{'.

And importing makeStyles should occur before import of useFormatter.

Reply:
Sure